python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/router-bgp.j2

```diff
@@ -1953,16 +2005,19 @@ router bgp {{ router_bgp.as }}
 {%             if vrf.address_family_ipv6.bgp.additional_paths.receive is arista.avd.defined(true) %}
          bgp additional-paths receive
 {%             endif %}
-{%             if vrf.address_family_ipv6.bgp.additional_paths.send.any is arista.avd.defined(true) %}
-         bgp additional-paths send any
-{%             elif vrf.address_family_ipv6.bgp.additional_paths.send.backup is arista.avd.defined(true) %}
-         bgp additional-paths send backup
-{%             elif vrf.address_family_ipv6.bgp.additional_paths.send.ecmp is arista.avd.defined(true) %}
-         bgp additional-paths send ecmp
-{%             elif vrf.address_family_ipv6.bgp.additional_paths.send.ecmp_limit is arista.avd.defined %}
-         bgp additional-paths send ecmp limit {{ vrf.address_family_ipv6.bgp.additional_paths.send.ecmp_limit }}
-{%             elif vrf.address_family_ipv6.bgp.additional_paths.send.limit is arista.avd.defined %}
-         bgp additional-paths send limit {{ vrf.address_family_ipv6.bgp.additional_paths.send.limit }}
+{%             if vrf.address_family_ipv6.bgp.additional_paths.send is arista.avd.defined %}
```

Comment:
for vrf.address_family_ipv6 you missed to add the j2 logic for neighbor additional path

Reply:
Added